src/screens/AddSiteFormScreen.js

```diff
@@ -300,30 +301,31 @@ class AddSiteFormScreen extends Component {
                         {location}
                     </Text>
 
-                    <Button
-                        large
-                        rounded={true}
-                        onPress={this.onClickIAmHere}
-                        buttonStyle={buttonStyle}
-                        icon={{name: 'bullseye', type: 'font-awesome'}}
+                    <LargeButton
                         title={here_now}
-                    >
-                    </Button>
+                        iconType={'font-awesome'}
+                        iconName={'bullseye'}
+                        iconColor={'white'}
+                        buttonStyleOverride={iAmHereButtonStyle}
+                        onPress={this.onClickIAmHere}
+                    />
 
                     <Text h3
                           style={headerTitle}
                     >- or -</Text>
 
-                    <FormLabel>{latitude}</FormLabel>
-                    <FormInput
+                    <Text style={labelStyle}>{latitude}</Text>
+                    <Input
+                        containerStyle={formInputStyle}
```

Comment:
`FormLabel` and `FormInput` were changed in the RNE upgrade. `FormLabel` was deprecated, and `FormInput` was renamed to `Input`. I had to do several style changes to get them to look good again. Feel free to keep tweaking.